Pet-Clinic-Frontend/src/pages/Login.tsx

```diff
@@ -1,51 +1,87 @@
 import { useState } from 'react'
-import './Login.css'
+import { useNavigate } from 'react-router-dom';  // must be install by command: npm list react-router-dom
+import './Login.css';
 
-
-function LoginForm() {
-  const [username, setUsername] = useState("")
+const Login = () => {
+  const [email, setemail] = useState("")
   const [password, setPassword] = useState("")
+  const [error, setError] = useState<string | null>(null);
+  const navigate = useNavigate();
+
+  const validateForm = () => {
+    if (!email || !password) {
+      setError("Email and password are required");
+      return false;
+    }
+
+    const emailRegex = /\S+@\S+\.\S+/;
+    if (!emailRegex.test(email)) {
+      setError("Must be a valid email address");
+      return false;
+    }
+    return true;
+  };
+
+  const sendToBackend = async (email: string, password: string) => {
+    try {
+      const response = await fetch('http://localhost:8080/login', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ email, password }),
+      });
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const data = await response.json();
+      if (data.success) {
```

Comment:
I dont understand this part, when we know that the data are successfull?

Reply:
If i understood question correctly. When you will be redirect to the landing page that means you are log in successfully. But anyway we/ I must tested with ready backend.